themes/openy_themes/openy_carnation/templates/paragraph/paragraph--banner.html.twig

```diff
@@ -59,19 +59,20 @@
     <div class="banner-cta-content container align-items-center m-auto py-5 text-center text-white">
       <div class="banner-cta-section">
 
-        {% if not content.field_prgf_headline.isEmpty() %}
+        {% if not paragraph.field_prgf_headline.isEmpty() %}
```

Comment:
this is not a W3C related refactoring. Let's avoid having unrelated changes for BC

Reply:
Yes, it is not related to W3C, but I noticed this bug (content.field_prgf_headline is an array and we can't perform isEmpty on it). Should it be done in separate PR?

Reply:
let it be here, thank you